blueprints/ember-electron/index.js

```diff
@@ -24,96 +21,63 @@ class EmberElectronBlueprint extends Blueprint {
-    // (including our own dependencies), and add an entry to package.json
-    // pointing the forge config to the config file that is copied from the blueprint files.
-
-    const { devDeps, exactDevDeps, deps } = require('electron-forge/dist/init/init-npm');
-
-    return task.run({
-      'save-dev': true,
-      verbose: false,
-      packages: devDeps
-    }).then(() => task.run({
-      // Dependencies we need to keep locked at specific versions
-      'save-dev': true,
-      'save-exact': true,
-      verbose: false,
-      packages: exactDevDeps
-    })).then(() => task.run({
-      // Production dependencies
-      save: true,
-      verbose: true,
-      packages: ['electron-protocol-serve@1.1.0', ...deps]
-    }));
+  _installElectronTooling(logger) {
+    logger.startProgress('Installing electron build tools');
+
+    return efImport({
+      updateScripts: false,
+      outDir: 'electron-out',
+    })
+      .then(() => this.addPackageToProject('devtron', '^1.4.0'))
+      // n.b. addPackageToProject does not let us save prod deps, so we task
+      .then(() => this.taskFor('npm-install').run({
+        save: true,
+        verbose: false,
+        packages: ['electron-protocol-serve@1.1.0'],
+      }))
+      .then(() => logger.message('Installed electron build tools'));
   }
 
-  _createResourcesDirectories() {
-    const platforms = ['', 'darwin', 'linux', 'win32'];
-    let promises = platforms.map((platform) => {
-      let gitKeepPath = path.join(this.options.project.root, 'ember-electron', `resources${platform ? '-' : ''}${platform}`, '.gitkeep');
-      return ensureFile(gitKeepPath);
-    });
+  _createResourcesDirectories(logger) {
+    const ensureFile = denodeify(fs.ensureFile);
 
-    return all(promises);
+    logger.startProgress('Creating ember-electron resource dirs');
+
+    let rootDir = this.options.project.root;
+    let promises = [null, 'darwin', 'linux', 'win32']
+      .map((platform) => platform ? `resources-${platform}` : 'resources')
+      .map((dirName) => path.join(rootDir, 'ember-electron', dirName))
+      .map((dirPath) => ensureFile(path.join(dirPath, '.gitkeep')));
+
+    return all(promises)
+      .then(() => logger.message('Created ember-electron resource dirs'));
   }
 
-  _ensurePackageJsonConfiguration() {
+  _ensurePackageJsonConfiguration(logger) {
+    const readJson = denodeify(fs.readJson);
+    const writeFile = denodeify(fs.writeFile);
+    const writeJson = denodeify(fs.writeJson);
+
     let packageJsonPath = path.join(this.project.root, 'package.json');
-    // We can't use this.project.pkg because we modified package.json by
-    // installing dependencies, and that won't be reflected in this.project.pkg
-    return readJson(packageJsonPath)
-      .then((packageJson) => {
-        packageJson.config = packageJson.config || {};
-        packageJson.config.forge = './ember-electron/.electron-forge';
+    let forgeConfigPath = './ember-electron/.electron-forge.js';
```

Comment:
Okay, I don't have a strong opinion, I'm just pointing out that one of our stated goals in moving it to the file where it is now was to make it more like the hopefully-familiar `.ember-cli`, which does not have an extension.